docs/email/mass-mailings-using-civimail.md

```diff
@@ -269,6 +269,8 @@ two options described next.
 Once you have defined all aspects of your mailing click on **Next >**
 (lower left) to advance to:
 
+Prior to CiviCRM version 5.27 Tracking options were turned on by default when creating a new mailing. From CiviCRM 5.27 onwards there is now a new setting at Administer -> CiviMail -> CiviMail Component Settings which will allow you to choose whether the Click Through or the Open tracking or both are turned on by default or not.
```

Comment:
This paragraph should be move up by one paragraph (so "Once you..." follows the new paragraph), and I would reword it slightly to say:
> Prior to CiviCRM version 5.27, tracking options were turned on by default when creating a new mailing. From CiviCRM 5.27 onwards, there are settings at Administer -> CiviMail -> CiviMail Component Settings which allow you to choose whether or not click-through and open tracking are turned on by default.

Reply:
Actioned. Merging!